mobile/src/full/res/xml/automotive_app_desc.xml

```diff
@@ -0,0 +1,4 @@
+<automotiveApp>
```

Comment:
Indeed, I guess I accidentally `git add`ed this leftover from a previous experiment. I'll remove it.